lib/workers/repository/process/limits.ts

```diff
@@ -6,33 +6,31 @@ import { PrState } from '../../../types';
 import { BranchConfig } from '../../common';
 
 export async function getPrHourlyRemaining(
-  config: RenovateConfig
+  config: RenovateConfig,
+  branches: BranchConfig[]
 ): Promise<number> {
   if (config.prHourlyLimit) {
     logger.debug('Calculating hourly PRs remaining');
-    const prList = await platform.getPrList();
-    const currentHourStart = DateTime.local().startOf('hour');
-    logger.debug(`currentHourStart=${String(currentHourStart)}`);
     try {
+      const branchList = branches.map(({ branchName }) => branchName);
+      const prList = await platform.getPrList();
+      const currentHourStart = DateTime.local().startOf('hour');
+      logger.debug(`currentHourStart=${String(currentHourStart)}`);
       const soFarThisHour = prList.filter(
         (pr) =>
           pr.sourceBranch !== config.onboardingBranch &&
+          branchList.includes(pr.sourceBranch) &&
```

Comment:
Would this mean that a PR that's created and automerged within the same hour won't be counted?

Reply:
If the intention is to find "our" PRs, then could instead filter on `branchPrefix`?